natgeo/mortar
# [feature/AEM-5364] Create CTA Button/Create Icon Component

This encompasses both stories as the AC is tightly coupled:
https://jira.nationalgeographic.com/browse/AEM-5406
https://jira.nationalgeographic.com/browse/AEM-5364

TO TEST:
Pull feature/AEM-5364
Run `gulp serve`
Change values in app/modules/button/docs.html && app/modules/icon/docs.html
Verify changes according to AC
